environment/environment_templates.go

```diff
@@ -311,6 +324,33 @@ func NewGethReorgHelmChart() *HelmChart {
 	}
 }
 
+// NewKafkaHelmChart creates new helm chart for kafka
+func NewKafkaHelmChart() *HelmChart {
+	valuesFilePath := filepath.Join(tools.ProjectRoot, "environment/charts/kafka/overrideValues.yaml")
+	overrideValues, err := chartutil.ReadValuesFile(valuesFilePath)
+	log.Debug().Interface("read values", overrideValues).Msg("read values")
```

Comment:
This debug log line still needed, or was it just for local debugging?

Reply:
only local debugging, will remove